blobstore/src/test/java/org/jclouds/blobstore/integration/internal/BaseContainerLiveTest.java

```diff
@@ -139,4 +177,17 @@ private void runCreateContainerInLocation(String payload, Location nonDefault) t
       }
    }
 
+   private void createContainer(String containerName, Location location) throws InterruptedException {
```

Comment:
Trying to factor out the container creation code (as there were additional assertions that the tests were doing).